src/pages/home.tsx

```diff
@@ -70,18 +70,19 @@ const HomePage = () => {
         <ContentBlock
           leftContent={
             <div>
-              <Typography.Title level={2}>Brewhaus Bakery</Typography.Title>
+              <Divider style={{borderColor:'rgb(55, 55, 55)', margin: 0, padding: '0 2rem 0 2rem'}}><Typography.Title level={1} id="welcome-to">Welcome To</Typography.Title></Divider>
```

Comment:
This has a lot of styles coming from a few directions-
1. The AntD Divider is providing some styling
2. Inline border color style
3. id tag

Makes me wonder if it would be good to standardize custom styles a bit. I used inline styles a lot because I'm lazy, but maybe its better to start using css files, and only do inlines with tailwind or something. Not a big deal right now, just something that popped in my head.

Reply:
yes, fully agree---not a fan of inline styling, will bloat up the code as more and more styling is added--i used inline here to quickly add some things but it seems very 'hacky' and not best practice

I think when you need to override antd styling, something like this maybe:

// whatever-page.tsx:
import styles from './index.css';
< ant-element className = {styles.class-to-add} />

// index.css:
.class-to-add {
   color: 'tomato';
   other-attributes: value;
   etc...
}

will work on cleaning this up